src/Illuminate/Routing/Console/stubs/controller.resource.stub

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace DummyNamespace;
+
+use Illuminate\Http\Request;
+
+use DummyRootNamespaceHttp\Requests;
+use DummyRootNamespaceHttp\Controllers\Controller;
+
+class DummyClass extends Controller
+{
+    /**
+     * Display a listing of DummyCollection
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $DummyCollection = DummyModel::all();
```

Comment:
This won't work. `DummyModel` is referring to the `DummyNamespace`, which is gonna be a controllers namespace. Include a `use DummyModel;` in the top.